lib/session.c

```diff
@@ -371,9 +371,6 @@ int rm_session_btrfs_clone_main(RmCfg *cfg) {
     }
 
 #else
-    (void)source;
-    (void)dest;
-    (void)read_only;
     rm_log_error_line(_("rmlint was not compiled with btrfs support."))
```

Comment:
Nitpicking: This will likely produce a warning that ``cfg is unused``. 
That's why we had those ``(void)`` casts for the old parameters in that place.